service/handlers_app_user.go

```diff
@@ -111,6 +114,14 @@ func SystemUserAssertionHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	// Fetch the account assertions from the store
+	accountAssertions, err := fetchAccountAssertions(model)
+	if err != nil {
+		logMessage("USER", "account-assertions", err.Error())
+		formatBooleanResponse(false, "account-assertions", "", "Error retrieving the account assertions from the store", w)
```

Comment:
In all the other cases where err is being returned have you verified it is not compromising security at all?